libs/tenant_size_model/src/svg.rs

```diff
@@ -6,13 +6,13 @@ const SVG_WIDTH: f32 = 500.0;
 struct SvgDraw<'a> {
     storage: &'a StorageModel,
     branches: &'a [String],
-    seg_to_branch: &'a [usize],
+    seg_to_branch: &'a [(usize, bool)],
```

Comment:
Didn't find a good way to propagate the info regarding whether a point is a lease.

Reply:
Sticking to this for now. We could do a better rewrite of visualization if we are switching to a new billing metric for storage. (using [`svg_fmt`](https://crates.io/crates/svg_fmt) crate as we are doing for the layer files visuals.